core/services/keystore/keys/ocr2key/key_bundle.go

```diff
@@ -26,6 +26,8 @@ type KeyBundle interface {
 	Unmarshal(b []byte) (err error)
 	Raw() Raw
 	OnChainPublicKey() string
+	// Decrypts ciphertext using the encryptionKey from an OCR2 OffchainKeyring
+	NaclBoxOpenAnonymous(ciphertext []byte) (plaintext []byte, err error)
```

Comment:
I'm not convinced that this is the best method name, compared to something obvious like "decrypt". Nacl and box are implementation details.

Reply:
This specific name was explicitly requested by Research.